app/dao/notifications_dao.py

```diff
@@ -116,6 +116,9 @@ def dao_create_notification(notification):
     if _should_record_notification_in_history_table(notification):
         db.session.add(NotificationHistory.from_original(notification))
 
+    if current_app.config['SEND_REPORTS']:
```

Comment:
Not sure the dao is the best place to put adding messages to the queue, as they are usually reserved for database actions only.

Is there a business object / method of notification which would be more suitable i.e. the method that calls this?